databases/business-sync/fragment-shoebox-snippet.js

```diff
@@ -0,0 +1,40 @@
+{
+  channels: getDocSyncChannels(doc, oldDoc),
+  typeFilter: function(doc, oldDoc) {
+    return createBusinessEntityRegex('shoeboxSnippet\\.[A-Za-z0-9_-]+\\.[A-Za-z0-9_-]+').test(doc._id);
```

Comment:
Comment to come on regex.

On the topic of `shoeboxSnippet`, I went with that because, as silly as it might sound, it ties in very closely with a concept/analogy that has been a motivator behind Kashoo pretty much since the beginning, and I think also ties in well with the 'smart inbox' idea.  I'm not overly attached to it, so if you feel strongly about it I'm open to suggestions.